akka-docs/src/main/paradox/typed/actors.md

```diff
@@ -427,3 +445,6 @@ the `Main` Actor terminates there is nothing more to do.
 Therefore after creating the Actor system with the `Main` Actor’s
 `Behavior` we can let the `main` method return, the `ActorSystem` will continue running and 
 the JVM alive until the root actor stops.
+
+@@include[includes.md](../includes/actors.md) { #actors-and-exceptions }
+See @ref:[supervision](../general/supervision.md).
```

Comment:
wouldn't it be better to have this include in supervision.md?
I don't think we should introduce too many concepts here.

Reply:
I mean typed/fault-tolerance.md

Reply:
It was in /actors.md, just moved here as well for new akka users. Will consider...

Reply:
Yes, agree.